### PR TITLE
Replace map via state object in Tabs

### LINKED ISSUE #455: Replace map via state object in Tabs

### PR DESCRIPTION
**Pull request by LamaEats**
Haven't visual changes, only api and inner login of Tabs component

## PR includes

- [x] Bug Fix

## Related issues

Resolve #455

### CODE REVIEW

File: src/components/Tabs.tsx [hunk 1]
```diff
@@ -10,27 +10,20 @@ import { TabsMenuDefault, TabsMenu } from './TabsMenu';
 interface TabsProps {
     layout: 'horizontal' | 'vertical';
     onChangeActiveTab?: (nextTab: string) => void;
+    activeTab?: string;
```
Comment (awinogradov) [inline src/components/Tabs.tsx:13]: It would be better to sync name with name on context. `active` is enough.

File: src/components/Tabs.tsx [hunk 2]
```diff
@@ -10,27 +10,20 @@ import { TabsMenuDefault, TabsMenu } from './TabsMenu';
 interface TabsProps {
     layout: 'horizontal' | 'vertical';
     onChangeActiveTab?: (nextTab: string) => void;
```
Comment (awinogradov) [inline src/components/Tabs.tsx:12]: Not in this diff, but `onChange` in enough.

File: src/stories/Tabs.stories.tsx [hunk 3]
```diff
@@ -30,35 +30,23 @@ export const Default: StoryFn<TabsProps> = (args) => {
     return (
         <div style={{ width: args.layout === 'vertical' ? '300px' : '500px', height: '200px' }}>
             <Tabs {...args}>
-                <Tab name="tab one" label={<TabLabel label="First" />} selected>
-                    <TabContentWrapper>
-                        <TabContent>content for first</TabContent>
-                    </TabContentWrapper>
+                <Tab name="1" label={<TabLabel label="First" />} selected>
```
Comment (awinogradov) [inline src/stories/Tabs.stories.tsx:33]: Why we need `selected` prop here if you add `active` to Tabs component?

**Review by awinogradov — CHANGES_REQUESTED** [review]

File: src/stories/Tabs.stories.tsx (hunk 3 above)
Comment (LamaEats) [reply src/stories/Tabs.stories.tsx:33]: I forgot to drop this prop

**Review by 9teen90nine — APPROVED** [review]

**Review by awinogradov — APPROVED** [review]

**Pull request merged**